keyboards/splitkb/kyria/keymaps/ArtFlag/readme.md

```diff
@@ -0,0 +1,27 @@
+# Simple workman layout
+
+## Features
+
+- Workman layout
+- semicolon and colon are flipped. Colon on tap, semicolon on double tap.
+
+<img src='layout.png'/>
```

Comment:
Please use markdown rather than html.

ALso, may need to link this to the actual image.

Reply:
updated for markdown, but i'm not sure what you mean about linking to the actual image?

Reply:
Have you looked at the preview?

Reply:
nothing wrong the link per se, but I just realized we gitignore png... so I removed the link, thanks for catching this.